### PR TITLE
DEVX-2366: Multiple extension points support

### PR DESCRIPTION
**Pull request by slitviachenko**
## Description
Changed the install.yml extension declaration to support multiple extension points.

## Related Issue
https://jira.corp.adobe.com/browse/DEVX-2366

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: install.yml [hunk 1]
```diff
@@ -1,9 +1,6 @@
-$schema: http://json-schema.org/draft-07/schema
-$id: https://adobe.io/schemas/app-builder-templates/1
```
Comment (slitviachenko) [inline install.yml:2]: $schema, $id should only be used in schema declarations

File: install.yml [hunk 2]
```diff
@@ -1,9 +1,7 @@
...
- 
 categories:
+  - helper-template
```
Comment (slitviachenko) [inline install.yml:2]: This is for DEVX-2457, "Exclude helper templates from a list of available for installation templates".

File: install.yml (hunk 1 above)
Comment (shazron) [reply install.yml:2]: No, these are identifiers that show what `install.yml` conforms to. For example, I can have `$id` that is a different version, that should be processed with a different schema version.

Comment (slitviachenko) [reply install.yml:2]: @shazron ,
It looks like I misunderstood the purpose of `$schema`, `$id` described on https://json-schema.org/learn/getting-started-step-by-step.html#starting

> The [$schema](https://json-schema.org/draft/2020-12/json-schema-core.html#rfc.section.8.1.1) keyword states that this schema is written according to a specific draft of the standard and used for a variety of reasons, primarily version control.
> The [$id](https://json-schema.org/draft/2020-12/json-schema-core.html#rfc.section.8.2.1) keyword defines a URI for the schema, and the base URI that other URI references within the schema are resolved against.

It mentions a schema declaration only.
I'll update [the current schema declaration](https://github.com/adobe/aio-lib-console-project-installation/blob/553e01b380bbdd47eb7c4a399e66eaf40185eb7e/schema/template.schema.json#L5-L27) to require `$schema` and `$id`.

I returned back `$schema` and `$id`.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/generator-add-action-generic",
-  "version": "0.2.3",
+  "version": "0.2.4",
```
Comment (shazron) [inline package.json:3]: everything else looks good, can we revert this version change though? it is automated by our publish action

**Review by shazron — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (slitviachenko) [reply package.json]: @shazron , ^^^ done

**Review by shazron — APPROVED** [review]

**Pull request merged**